src/app/components/login/login.component.css

```diff
@@ -1,125 +1,69 @@
-    background: -webkit-linear-gradient(rgb(202, 190, 228), rgba(44, 41, 214, 0.753));
-    background-clip: border-box;
-    -webkit-background-clip: text;
-    -webkit-text-fill-color: #cfcfe33d;
-    font-family: Arial, Helvetica, sans-serif;
-    text-shadow: 4px 4px 2px rgba(45, 45, 89, 0.52);
-}
-
-@media screen and (max-width: 1242px) {
-    .container h1 {
-        font-size: 50px;
-        margin-top: 10vw;
-        margin-bottom: 10vw;
-        background: -webkit-linear-gradient(rgb(202, 190, 228), rgba(44, 41, 214, 0.753));
-        background-clip: border-box;
-        -webkit-background-clip: text;
-        -webkit-text-fill-color: #cfcfe33d;
-        font-family: Arial, Helvetica, sans-serif;
-        text-shadow: 4px 4px 2px rgba(45, 45, 89, 0.52);
-    }
-}
-
-.form {
-    display: flex;
-    justify-content: center;
-    min-width: 200px;
-    width: 25vw;
-    height: 30vh;
-    border: 10px outset rgba(1, 8, 8, 0.116);
-}
-
-.form button {
-    align-self: center;
-}
-
-mat-label {
-    display: flex;
-    justify-content: center;
-    color: rgba(255, 255, 255, 0.473);
-}
-
-mat-form-field {
-    display: flex;
-    justify-content: center;
-}
-
-.form h3 {
-    font-size: 1vw;
-    display: flex;
-    justify-content: center;
-    align-items: center;
-    color: red;
-    margin-bottom: 15px;
-    text-shadow: 4px 4px 2px rgba(0, 0, 0, 0.52);
-}
-
-@media screen and (max-width: 1242px) {
-    .form h3 {
-        font-size: 10px;
-        display: flex;
-        justify-content: center;
-        align-items: center;
-        color: red;
-        text-shadow: 4px 4px 2px rgba(0, 0, 0, 0.52);
-    }
-}
\ No newline at end of file
+        .my-container {
+            height: 100vh;
+            background: linear-gradient(277deg, #1946db, #ca2580, #1946db);
+            background-size: 400% 400%;
+            -webkit-animation: AnimationName 19s ease infinite;
+            -moz-animation: AnimationName 19s ease infinite;
+            animation: AnimationName 19s ease infinite;
+        }
+        
+        @-webkit-keyframes AnimationName {
+            0% {
+                background-position: 0% 50%
+            }
+            50% {
+                background-position: 100% 51%
+            }
+            100% {
+                background-position: 0% 50%
+            }
+        }
+        
+        @-moz-keyframes AnimationName {
+            0% {
+                background-position: 0% 50%
+            }
+            50% {
+                background-position: 100% 51%
+            }
+            100% {
+                background-position: 0% 50%
+            }
+        }
+        
+        @keyframes AnimationName {
+            0% {
+                background-position: 0% 50%
+            }
+            50% {
+                background-position: 100% 51%
+            }
+            100% {
+                background-position: 0% 50%
+            }
+        }
+        
+        .errormessaje {
```

Comment:
engleza, iar pentru faza cu color red, cred ca ai clasa text-danger in bootstrap